cmd/vpn_client/app/app.go

```diff
@@ -56,23 +56,26 @@ func NewCommand() *cobra.Command {
 
 func vpnConfig(log logr.Logger, cfg config.VPNClient) openvpn.ClientValues {
 	v := openvpn.ClientValues{
-		Device:               constants.TunnelDevice,
-		IPFamily:             cfg.PrimaryIPFamily(),
-		ReversedVPNHeader:    cfg.ReversedVPNHeader,
-		Endpoint:             cfg.Endpoint,
-		OpenVPNPort:          cfg.OpenVPNPort,
-		VPNClientIndex:       cfg.VPNClientIndex,
-		IsShootClient:        cfg.IsShootClient,
-		IsHA:                 cfg.IsHA,
-		SeedPodNetwork:       cfg.SeedPodNetwork.String(),
-		SeedPodNetworkMapped: constants.SeedPodNetworkMapped,
+		Device:            constants.TunnelDevice,
+		IPFamily:          cfg.PrimaryIPFamily(),
+		ReversedVPNHeader: cfg.ReversedVPNHeader,
+		Endpoint:          cfg.Endpoint,
+		OpenVPNPort:       cfg.OpenVPNPort,
+		VPNClientIndex:    cfg.VPNClientIndex,
+		IsShootClient:     cfg.IsShootClient,
+		IsHA:              cfg.IsHA,
+		SeedPodNetwork:    cfg.SeedPodNetwork.String(),
 	}
 	vpnSeedServer := "vpn-seed-server"
 
 	if len(strings.Split(cfg.IPFamilies, ",")) == 2 {
 		v.IsDualStack = true
 	}
 
+	if !cfg.IsHA && cfg.SeedPodNetwork.IsIPv4() {
+		v.SeedPodNetwork = constants.SeedPodNetworkMapped
+	}
```

Comment:
What happens in a dual-stack cluster?

Reply:
dual-stack shoots will work fine as long as the seed is ipv4.
the value for `v.SeedPodNetwork` is used in [openvpn client config](https://github.com/gardener/vpn2/blob/master/pkg/openvpn/assets/client-config.template#L48) to add a v4 route back to the seed. 

dual-stack seeds aren't supported yet and the code in vpnconfig is unchanged with this PR. my change only makes sure that we don't try to call `/sbin/ip route replace` with a v6 address accidentally. the main purpose of the change is to change the original v4 seed pod range to the NATed range of 241/8 so that it can route back through the tunnel.